conjure-java-core/src/integrationInput/java/com/palantir/product/ErrorServiceBlocking.java

```diff
@@ -0,0 +1,386 @@
+                Preconditions.checkArgumentNotNull(value, "value cannot be null");
+            }
+        }
+
+        final class InvalidArgument extends ConjureErrors.BaseEndpointError<TestErrors.InvalidArgumentParameters>
+                implements TestMultipleErrorsAndPackagesResponse {
+            @JsonCreator
+            InvalidArgument(
+                    @JsonProperty("errorCode") String errorCode,
+                    @JsonProperty("errorInstanceId") String errorInstanceId,
+                    @JsonProperty("parameters") TestErrors.InvalidArgumentParameters parameters) {
+                super(errorCode, TestErrors.INVALID_ARGUMENT.name(), errorInstanceId, parameters);
+            }
+        }
+
+        final class NotFound extends ConjureErrors.BaseEndpointError<TestErrors.NotFoundParameters>
+                implements TestMultipleErrorsAndPackagesResponse {
+            @JsonCreator
+            NotFound(
+                    @JsonProperty("errorCode") String errorCode,
+                    @JsonProperty("errorInstanceId") String errorInstanceId,
+                    @JsonProperty("parameters") TestErrors.NotFoundParameters parameters) {
+                super(errorCode, TestErrors.NOT_FOUND.name(), errorInstanceId, parameters);
+            }
+        }
+
+        final class DifferentNamespace extends ConjureErrors.BaseEndpointError<ConjureErrors.EmptyErrorParameters>
+                implements TestMultipleErrorsAndPackagesResponse {
+            @JsonCreator
+            DifferentNamespace(
+                    @JsonProperty("errorCode") String errorCode,
+                    @JsonProperty("errorInstanceId") String errorInstanceId) {
+                super(
+                        errorCode,
+                        EndpointSpecificTwoErrors.DIFFERENT_NAMESPACE.name(),
+                        errorInstanceId,
+                        ConjureErrors.EMPTY_ERROR_PARAMETERS_INSTANCE);
+            }
+        }
+
+        final class DifferentPackage extends ConjureErrors.BaseEndpointError<ConjureErrors.EmptyErrorParameters>
+                implements TestMultipleErrorsAndPackagesResponse {
+            @JsonCreator
+            DifferentPackage(
+                    @JsonProperty("errorCode") String errorCode,
+                    @JsonProperty("errorInstanceId") String errorInstanceId) {
+                super(
+                        errorCode,
+                        com.palantir.another.EndpointSpecificErrors.DIFFERENT_PACKAGE.name(),
+                        errorInstanceId,
+                        ConjureErrors.EMPTY_ERROR_PARAMETERS_INSTANCE);
+            }
+        }
+    }
+
+    sealed interface TestEmptyBodyResponse
+            permits TestEmptyBodyResponse.Success, TestEmptyBodyResponse.InvalidArgument {
+        record Success() implements TestEmptyBodyResponse {
+            @JsonCreator
+            public static Success create() {
+                return new Success();
+            }
```

Comment:
The other success variants don't appear to be annotated -- is this method necessary?

Reply:
We only need it when deserializing empty body responses so that the reflection code in Dialogue's `JacksonEmptyContainerLoader` [(src)](https://github.com/palantir/dialogue/pull/2443/files#diff-8d97a05b5c1cad8e5e6afe3d1d1b8e6a3fbc5f106fd7417f6282244184422142) can invoke this method to construct an empty instance.

Reply:
yeah that works too. Avoids us generating these ctors in the records.

Reply:
Updated the [dialogue PR here](https://github.com/palantir/dialogue/pull/2443/files#diff-8d97a05b5c1cad8e5e6afe3d1d1b8e6a3fbc5f106fd7417f6282244184422142R169) to invoke the 0-parameter record constructor.